packages/neutron-sdk/src/interchain_queries/register_queries.rs

```diff
@@ -68,6 +70,120 @@ pub fn register_balance_query_msg(
     )
 }
 
+/// Creates a message to register an Interchain Query to get total supply on remote chain for particular denom
+///
+/// * **connection_id** is an IBC connection identifier between Neutron and remote chain;
+/// * **denom** denomination of the coin for which you want to get balance;
+/// * **update_period** is used to say how often the query must be updated.
+pub fn register_bank_total_query_msg(
+    deps: DepsMut<InterchainQueries>,
+    env: Env,
+    connection_id: String,
+    denom: String,
```

Comment:
Maybe we can support multiple denoms in one ICQ (`denom: Vec<String>`)?
Cause `TotalSupply` response struct already supports multiple coins